src/Parser.js

```diff
@@ -69,7 +69,7 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 					case 0: case 125: scanning = 0
 					// ;
 					case 59 + offset:
-						if (property > 0)
+						if (property > 0 && (strlen(characters) - length))
```

Comment:
the idea behind the fix is that it checks if anything has been consumed for the value, if the diff is 0 then no - nothing has been consumed